### DISCUSSION TITLE
Boundary conditions for scalar.

### DISCUSSION THREAD

**Discussion by sanjeev784** [xcompact3d/Incompact3d#250] [Q&A] [answered]
alpha_sc, beta_sc, g_sc are used only when iimplicit > 0. Can anybody let me know where i can change alpha and beta for explicit method ?

**Comment by slaizet** [comment, marked as answer]
Hi,
You can change alpha_sc, beta_sc for an explicit time advancement scheme. This can be done in the boundary conditions subroutine in the relevant BC-XXX file (depending on your flow configuration). This has not been done yet, but it is possible.
Cheers,
Sylvain
